src/psyclone/transformations.py

```diff
@@ -1124,8 +1135,10 @@ def apply(self, node):
         # can be run in parallel
         colour_loop = node.__class__(parent=colours_loop, loop_type="colour")
         colour_loop.field_space = node.field_space
+        colour_loop._field_name = node._field_name
         colour_loop.iteration_space = node.iteration_space
         colour_loop.set_lower_bound("start")
+        colour_loop._kern = node._kern
```

Comment:
Again, a protected access. Create an accessor  function?

Reply:
Actually, it turned out that only the `dynamo0p3.Kern` class had this `_kern` property. I've moved it into the base class and added getter/setter methods.

Reply:
Great.